src/Service.Host/client/src/components/OverbooksSearch.js

```diff
@@ -1,15 +1,25 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { collectionSelectorHelpers, Page, Typeahead } from '@linn-it/linn-form-components-library';
+import {
+    collectionSelectorHelpers,
+    Page,
+    Typeahead,
+    utilities
+} from '@linn-it/linn-form-components-library';
 import Typography from '@mui/material/Typography';
 import Grid from '@mui/material/Grid';
-
+import ModeEditIcon from '@mui/icons-material/ModeEdit';
+import EditOffIcon from '@mui/icons-material/EditOff';
+import Tooltip from '@mui/material/Tooltip';
 import purchaseOrdersActions from '../actions/purchaseOrdersActions';
+import purchaseOrdersApplicationStateActions from '../actions/purchaseOrdersApplicationStateActions';
 import history from '../history';
 import config from '../config';
 
 function OverbooksSearch() {
     const dispatch = useDispatch();
+    useEffect(() => dispatch(purchaseOrdersActions.fetch()), [dispatch]);
+    useEffect(() => dispatch(purchaseOrdersApplicationStateActions.fetchState()), [dispatch]);
```

Comment:
yeah, I think you can just do purchaseOrdersActions.fetchState() here, and remove line 21 and delete the extraneous  purchaseOrdersApplicationStateActions.js

Reply:
Ah ok! Cheers for checking that.

Just sorting a GIT issue I'm having atm as my test isn't pushing. Will have that sorted in a sec.

Reply:
Just a general Q but we have StateAPI actions and FetchAPI actions - what's the relationship here? Is there a benefit to using StateAPI actions over FetchAPI in select circumstances? Will have a look at the common code but just wanted to check in first.

Reply:
only if you have no need for fetch api. they started separate but got added to fetch and update for convenience
 from memory